doc/runbooks/snapshot_recovery_azure.md

```diff
@@ -0,0 +1,207 @@
+# Recovering snapshot backups on Microsoft Azure
```

Comment:
Does anything actually link to this runbook?

The google cloud script can be reached from docs.pgbarman.org so I think we should add a link to this runbook in a similar way.